automated_packaging/update_documentation.pl

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/perl
+use lib '/usr/local/bin';
+use common_functions;
+
+$NEW_VERSION = $ARGV[0];
+$OLD_VERSION = $ARGV[1];
+
+my $github_token = get_and_verify_token();
+
+# Necessary to create unique branch
+$curTime = time();
+
+`git checkout master`;
+`git checkout -b master-$NEW_VERSION-$curTime`;
+
+$new_release_minor_version = substr( $NEW_VERSION, 0, 3 );
+$old_release_minor_version = substr( $OLD_VERSION, 0, 3 );
+$old_release_minor_version_escape_dot = $old_release_minor_version;
+$old_release_minor_version_escape_dot =~ s/\./\\./g;
+
+# Update conf.py file
+`sed -i 's/$old_release_minor_version_escape_dot/$new_release_minor_version/g' conf.py`;
```

Comment:
No need to update this. It is updated by @begriffs when we switch to new documentation version.